docker-compose.yml

```diff
@@ -51,26 +54,43 @@ services:
     depends_on:
       - jaeger-all-in-one
       - zipkin-all-in-one
+
+  carbon-aware-sdk:
+    image: carbon_aware:v1
+    ports:
+      - "80:80"
+    container_name: carbon-aware-sdk
+    environment:
+      - CarbonAwareVars__CarbonIntensityDataSource=WattTime
+      - WattTimeClient__Username=gsf_iamdebojyoti
+      - WattTimeClient__Password=UNI6g!tztB*+2Ls(7
```

Comment:
@iamdebojyoti I don't think you want to check in your WattTime password - can you drop this field from the docker compose and maybe replace it with something like a required environment variable `${WATTTIME_CLIENT_PASSWORD:?err}` so that it'll fail if the user doesn't specify it manually or through a .env file?